src/root-redirect.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE HTML>
+<html lang="en-US">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="refresh" content="1;url=/#LATEST_VERSION#/docs/">
```

Comment:
@aju maybe you thought about this one? That I should redirect to /version/docs/basics.html instead of /version/docs? /version/docs, for some reason, opens basics.html file by default